packages/react-strict-dom/tests/__snapshots__/css-test.native.js.snap-native

```diff
@@ -35,7 +35,11 @@ exports[`properties: custom property css.defineVars: tokens 1`] = `
 }
 `;
 
-exports[`properties: custom property handles undefined variables 1`] = `{}`;
+exports[`properties: custom property handles undefined variables 1`] = `
+{
+  "style": {},
```

Comment:
Try to avoid empty styles

Reply:
Going to keep returning the empty object from the css function, as it's created in the call anyway. But to preserve the auto-wrap text snapshots, I'll delete empty styles for that component.